ChangeLog

```diff
@@ -1,3 +1,11 @@
+2015-05-27  Jim Shaver <dcypherd@gmail.com>
+
+	* OpenSSL/SSL.py, : Add ``get_protocol_version()`` and 
+	  ``get_protocol_version_name()`` to Connection
```

Comment:
I’d like Connection to be in `` too.  Also please add some periods after sentences. :)